ddtrace/_trace/trace_handlers.py

```diff
@@ -153,6 +154,43 @@ def _maybe_start_http_response_span(ctx: core.ExecutionContext) -> None:
         )
 
 
+def _use_html(headers) -> bool:
+    """decide if the response should be html or json.
+
+    Add support for quality values in the Accept header.
+    """
+    ctype = headers.get("Accept", headers.get("accept", ""))
+    if not ctype:
+        return False
+    html_score = 0.0
+    json_score = 0.0
+    ctypes = ctype.split(",")
+    for ct in ctypes:
+        if len(ct) > 128:
+            # ignore long (and probably malicious) headers to avoid performances issues
+            continue
+        m = re.match(r"([^/;]+/[^/;]+)(?:;q=([01](?:\.\d*)?))?", ct.strip())
+        if m:
+            if m.group(1) == "text/html":
+                html_score = max(html_score, min(1.0, float(1.0 if m.group(2) is None else m.group(2))))
+            elif m.group(1) == "text/*":
+                html_score = max(html_score, min(1.0, float(0.2 if m.group(2) is None else m.group(2))))
+            elif m.group(1) == "application/json":
+                json_score = max(json_score, min(1.0, float(1.0 if m.group(2) is None else m.group(2))))
+            elif m.group(1) == "application/*":
+                json_score = max(json_score, min(1.0, float(0.2 if m.group(2) is None else m.group(2))))
+    return html_score > json_score
```

Comment:
nit: In the case of a tie (not sure how likely/unlikely this is), we're defaulting to using `application/json`?

Reply:
Yes, that was already the case before if there was nothing in the accept header, we were defaulting to json. 
Now, it's the same, in case of a tie.